examples/prisma/.gitignore

```diff
@@ -0,0 +1,9 @@
+node_modules
```

Comment:
@juwiragiye Can you please delete the Prisma example from this branch/PR?

Reply:
@MichaelDeBoey I successfully deleted prisma example from this branch/PR